xeno/shield_removal.md

```diff
@@ -0,0 +1,94 @@
+
+Shields are unintuitive and frustrating. Shields have no visual cues or indicators to marines, making it hard and frustrating to predict/anticipate. Runners can tank PBs etc and it makes xeno damage sprites not do the job they should. New players can have fights with xenos that are almost 50% stronger than as they expect, making it disheartening for them to learn that this xeno had more health with no visual cue at all. While the audio cue mitigates this in part, there’s nothing in-game that clues people into what the audio cue actually represents, and going any further to make shields more visible would impact the game’s overall aesthetic & visual clarity.
+
+They’re often a balance nightmare. Shields mess with the AP economy majorly. They make the optimal ammo for a xeno constantly change depending on what health type they have (shield or health) and as a result they'll always impact how much total damage you can do against a xeno due to this. In addition to this, shields play a large role in power-creep, as you have to account for the large amount of shields Xenos can acquire at any one time on top of their existing health and armour pools, as well as the inconsistent “types” of shielding, i.e. Crusher shield having the property of reducing incoming damage, while something like Warden shield does not.
+
+They are too much of a buff. Shields are long term health buffs that don't expire over time, only with use. See - shamans giving the entire hive big shields on hijack, wardens being able to do the funny with shields i.e. constantly shielding through overwatch or stacking them over and over on a frontliner, etc.
+
+Shields handhold xenos - they provide an extra health buffer that many xenos can end up relying on for engagements, buffing the side's overall strength but limiting their individual strength (think like hugger combat). Removing shields would allow us to buff xenos to consistently have the right strength for engagements instead of this varying wildly between them. Shields also contribute to a “pocketing” meta wherein one or several Xenos that can grant shields will constantly be supplying a single player with a near-unending supply of what is effectively better health, drastically tipping the scales in a way the game isn’t really built to account for.
+
+Other buffs are more interesting. While shields may be strong, they are boring - all they do is just add extra health and let the xenos stick around on the front for longer. Other mechanics, such as speed boosts, clarity stacks (or even tgmc style increased evasion chance) would be more engaging, unique, and some would allow for more skillful use with them (i.e. a shield boost or a proc’ed next slash with a time expiry, like sentinel.) At the end of the day shields are lazy design that would be more interesting if replaced with a wider range of abilities.
```

Comment:
This doesn't need to be in the design doc itself if you are overhauling shields. Integrate these into a short list of non-goals that can be included. The explanation for why you are overhauling shields can be included in the PR description, but it's not necessary for future devs to know the problems of past shields, only what needs to be done to avoid having these problems happen again

Reply:
moved down and integrated into goals + non goals